x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/transforms/DataFrameIndexerTransformStats.java

```diff
@@ -57,60 +55,46 @@
         LENIENT_PARSER.declareLong(constructorArg(), SEARCH_FAILURES);
     }
 
-    private final String transformId;
-
     /**
      * Certain situations call for a default transform ID, e.g. when merging many different transforms for statistics gather.
```

Comment:
nit: these comments aren't true anymore

Reply:
I removed the method as well as the comment as after the other changes I don't think it was adding much over calling the default constructor.